### ISSUE TITLE
[IDEA] GettingStarted Tiddler in Empty.html

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#6870]
The current GettingStarted Tiddler is like below

```
Welcome to ~TiddlyWiki and the ~TiddlyWiki community

Before you start storing important information in ~TiddlyWiki it is vital to make sure that you can reliably save changes. See https://tiddlywiki.com/#GettingStarted for details

!! Set up this ~TiddlyWiki

```

I would recommend to add:

- link to Talk forum
- link to links.tiddlywiki.org

**Comment by amreus** [comment]
Agree, and it would be handy to see the `<<version>>` somewhere on a new wiki.

**Comment by twMat** [comment]
Apropos improving the GettingStarted tiddler in empty I also proposed #1344 ... a while ago hmmm... but it is a bit beyond my level to create a PR for it... :-(

@amreus
>Agree, and it would be handy to see the <<version>> somewhere on a new wiki.

To not go OT, I posted #6873 that partly deals with this. Please chime in there if you agree.

**Comment by pmario** [comment]
hmmmm,  https://links.tiddlywiki.org/ .. As a new user I wouldn't want to be directed to the links page. 

I'll create a new issue about the links-page after searching for existing issues.

**Comment by twMat** [comment]
> As a new user I wouldn't want to be directed to the links page.

What do you mean here? Do you not like the idea of external links there or do you not think new users should be exposed to the links site? If it is the latter, why? As a user I would definitely want to know right away where I can find plugins so I think this is a really good place. The only problem is that the GettingStarted tiddler "disappears" after you've set it so we should, IMO, also show the link to links elsewhere.

**Comment by pmario** [comment]
I think there is room for improvement. If you are at the links-page, it needs to be much more obvious, where to click, that you can go to the site / plugin you wanted to find. ... I need to test all the tiny buttons, that have no visual feedback on hover ...

See: 

>I'll reuse: [Idea] Update links.tiddlywiki.com with community links - summarize Community and send users to links page https://github.com/Jermolene/TiddlyWiki5/issues/6648

**Comment by twMat** [comment]
> I think there is room for improvement. If you are at the links-page,

OK, so you don't actually dislike the idea to include a link to the links page in GettingStarted - just that the page is not ready so "let's wait".

**Comment by Jermolene** [comment]
As to the links page, the way that we'll improve it is by featuring it prominently so that it gets eyeballs.
